rl4co/models/zoo/deepaco/antsystem.py

```diff
@@ -93,9 +96,9 @@ def one_step(self, td: TensorDict, env: RL4COEnvBase):
             reward: The reward achieved by the algorithm.
         """
         # sampling
-        actions, reward = self._sampling(td, env)
+        td, env, actions, reward = self._sampling(td, env)
         # local search, reserved for extensions
-        actions, reward = self._local_search(actions, reward)
+        actions, reward = self._local_search(td, env, actions, reward)
```

Comment:
[Minor] in case we don't want to do local search, I think we could set a parameter for DeepACO such as `use_local_search` (I would say set to "False" by default) ; then once the `env` class has the local search option, the env could raise an assertion error if it is not implemented (e..g [here](https://github.com/ai4co/rl4co/pull/140/files#r1528357231))